mysql-test/suite/rpl/t/rpl_warn_statement.test

```diff
@@ -0,0 +1,2 @@
+--source include/have_binlog_format_statement.inc
+--source rpl_warn.inc
```

Comment:
These tests would be better suited in the `binlog` suite (`./mysql-test/suite/binlog/t`). The `rpl` suite is used for tests which use a `master` and `slave`.  Then since it is a mysqlbinlog specific test, there is another convention of using the test prefix `binlog_mysqlbinlog_*`.

Reply:
Great idea, I will move them.